src/http/matcher/mod.rs

```diff
@@ -569,7 +593,11 @@ impl HttpMatcher {
     }
 }
```

Comment:
Still to be done, add the following methods: `::custom`, `::and_custom`, `::or_custom`

Reply:
Done.